src/board.c

```diff
@@ -319,55 +270,12 @@ void Board_GPIO_Init(void) {
 #ifndef TEST_HARDWARE
     Chip_GPIO_Init(LPC_GPIO);
 
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED0);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED1);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, LED2);
-    /* Select function PIO1_3 */ 
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_LED2, IOCON_FUNC1);	
-    Chip_GPIO_SetPinDIRInput(LPC_GPIO, CTR_SWTCH);
 #ifdef FSAE_DRIVERS
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, FSAE_FAULT_GPIO);
-    /* Select function PIO3_0 */
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_FSAE_FAULT_GPIO, IOCON_FUNC0); 
-    
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, FSAE_CHARGE_ENABLE_GPIO);
-    /* Select function PIO3_2 */
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_FSAE_CHARGE_ENABLE_GPIO, IOCON_FUNC0); 
-
-    // configure fan 1 pin
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, FSAE_FAN_1_PIN);
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_FSAE_FAN_1_PIN, 
-            (IOCON_FUNC2 | IOCON_MODE_INACT));
-    Chip_GPIO_SetPinState(LPC_GPIO, FSAE_FAN_1_PIN, false);
```

Comment:
@jatron in old code, pin state is initialized even though it's a timing bit - I assume this was a bug?